scipy/stats/tests/test_discrete_basic.py

```diff
@@ -271,3 +271,13 @@ def test_methods_with_lists(method, distname, args):
     npt.assert_allclose(result,
                         [dist.pmf(*v) for v in zip(z, *p2, loc)],
                         rtol=1e-15, atol=1e-15)
+
+
+def test_cdf_gh13280_regression():
+    vals0 = stats.hypergeom.cdf(np.arange(5), 3, 3, 4)
+    expected0 = np.array(5*[np.nan], dtype='d')
```

Comment:
This is great. Very simple, but a big improvement.
Does `dtype='d'` need to be specified?

Reply:
@mdhaber: no, `np.array` will infer float64 from the occurrence of `nan` in the input data.